Dockerfile

```diff
@@ -58,6 +58,26 @@ RUN \
   apt-get -qq autoclean && \
   apt-get -qq clean && \
   rm -rf /var/lib/apt/lists/* && \
+  rm -rf /var/log/apt/* && \
+  rm -rf /var/log/dbconfig-common/ && \
+  rm /var/log/alternatives.log && \
+  rm /var/log/dpkg.log && \
+  rm /var/log/faillog && \
+  rm /var/log/lastlog && \
+  rm -rf /var/cache/* && \
+  # Remove Debian documentation
+  rm -rf /usr/share/doc/* && \
+  rm -rf /usr/share/doc-base/ && \
+  # Remove useless .desktop file entries
+  rm /usr/share/applications/*.desktop && \
+  rm -rf /usr/share/apps/konqueror && \
+  rm -rf /usr/share/apps/konsole && \
```

Comment:
Do these locations/files really get created or is this just a "most complete" list of dirs to remove after installation?

Reply:
Yes, they get created
I would recommend you to use the tool dive to have a look ;)